concent-builder/migrate-db.yml

```diff
@@ -14,16 +14,6 @@
               --project "{{ gke.project }}"                  \
               --zone    "{{ gke.zone }}"
 
-        - name:    Delete outdated secrets and config maps required by jobs
-          command: "{{ build_dir }}/concent-deployment/kubernetes/build/delete-job-dependencies.sh"
-          args:
-            chdir: "{{ build_dir}}/concent-deployment/kubernetes/build/"
-
-        - name:    Create up to date secrets and config maps required by jobs
-          command: "{{ build_dir }}/concent-deployment/kubernetes/build/create-job-dependencies.sh {{ cluster_type }}"
-          args:
-            chdir: "{{ build_dir}}/concent-deployment/kubernetes/build/"
-
         - name:    Migrate the database used by concent-api
           command: kubectl create \
```

Comment:
maybe https://docs.ansible.com/ansible/2.5/modules/k8s_raw_module.html here?

Reply:
This should be separate issue to research and change in all playbooks. @cameel What are you think about it?

Reply:
I think we want to stick to our scripts because we'll be replacing it with Helm soon anyway.